BookStore/src/main/java/ro/utcn/fulgadan/BookStore/business/validator/Validator.java

```diff
@@ -0,0 +1,6 @@
+package ro.utcn.fulgadan.BookStore.business.validator;
+
+public interface Validator<T> {
```

Comment:
It is nice that you implemented such validator mechanism. For future reference, spring comes with the Hibernate Validator built in, so you can also use that in the future: https://www.baeldung.com/spring-boot-bean-validation